frontend/components/panels/NetProfitCheckPanel.vue

```diff
@@ -22,9 +22,19 @@ export default Vue.extend({
             type: Object as Vue.PropType<BigNumber>,
             default: undefined,
         },
+        negativeGrossProfit: {
+            type: Boolean,
+            default: false,
+        },
```

Comment:
I would suggest to either: not provide it here at all or to actually combine net and gross profits into a single `ProfitCheckPanel` (and therefore provide both values)

Reply:
I would avoid this as it would result in me needing to wrap both panels in a div. This would then mess up the rounding of the panels:

![Screenshot 2022-04-08 at 11 47 19](https://user-images.githubusercontent.com/30908158/162411299-9334f067-11e7-4ef2-89f4-53b7b3c238a3.png)

Reply:
No, I didn't mean to wrap two panels into one component, but to merge them into one panel which is: 0) `inactive` if there is no wallet 1) `incorrect` when net profit is negative 2) `notice` when gross profit is negative 3) `correct` when gross profit is positive. Would that make sense?